core/iwasm/common/wasm_runtime_common.c

```diff
@@ -4212,6 +4212,11 @@ static V128FuncPtr invokeNative_V128 = (V128FuncPtr)(uintptr_t)invokeNative;
           || defined(BUILD_TARGET_RISCV64_LP64) */
 #endif /* end of defined(_WIN32) || defined(_WIN32_) */
 
+#if defined(__GNUC__)
+#pragma GCC push_options
+#pragma GCC optimize("O0")
```

Comment:
Why need to set `O0` for this function? Its code size may be large for O0, and the performance may be greatly impacted.